app/views/candidate_interface/gcse_details/edit_details.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :title, page_title('Grade and award year') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_gcse_details_edit_type_path, 'Back to application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @application_qualification, url: candidate_interface_gcse_details_update_details_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder, method: :patch do |f| %>
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8">
+        <h1 class="govuk-fieldset__heading"><%= heading_for_gcse_edit_details(@subject) %></h1>
+      </legend>
+
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_text_field :grade, label: { text: 'Enter your qualification grade', size: 'm' } %>
```

Comment:
In the prototype, we use the width of 10. This can easily be done by passing in `width: 10`.

![image](https://user-images.githubusercontent.com/42817036/68209502-6d609180-ffcb-11e9-861e-38c7afd2b001.png)